Week5/mission/.env

```
@@ -0,0 +1 @@
+VITE_SERVER_API_URL= http://localhost:8000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)